app/views/booking_requests/customer.html.erb

```diff
@@ -0,0 +1,39 @@
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
```

Comment:
I'll leave it up to you. Not sure any real value in listing each file, especially as chances are you will need to list ever file.